pom.xml

```diff
@@ -132,6 +134,11 @@
                         <failOnMissingHeader>true</failOnMissingHeader>
                         <failOnNotUptodateHeader>true</failOnNotUptodateHeader>
                         <licenseName>apache_v2</licenseName>
+                        <!-- added to prevent license plugin from complaining 
+                            about missing headers in generated sources -->
```

Comment:
Makes it not more sense to exclude the target folder?

Reply:
absolutely